### PR TITLE
fix(rollingPush): reset start time

### PR DESCRIPTION
**Pull request by emjburns**
We've seen that with rolling push + a specified wait time, this stage is only waiting after the first termination. It _seems_ like start time is never getting reset, and therefore all following wait stages immediately complete because they think they started at the same time as the first wait stage.

This PR resets start time to the lowest time possible. If it sees that time in the running stage, it resets the time to the current time.

@robfletcher - does this seem like a reasonable fix? I'd like to remove the variable from the context entirely (that seems preferable to `Instant.MIN`), but I'm not sure if that's possible.

### CODE REVIEW

**Comment by ajordens** [comment]
It might be neat to have a contrived test in `WaitTaskSpec` that executes a `CheckForRemainingTerminationsTask` followed by a `WaitTask` ... to try and catch a situation like this in the future.

Will leave the value of that up to you @emjburns.

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/rollingpush/CheckForRemainingTerminationsTask.groovy [hunk 1]
```diff
@@ -35,7 +37,8 @@ class CheckForRemainingTerminationsTask implements Task {
 
     return new TaskResult(ExecutionStatus.REDIRECT, [
       skipRemainingWait: false,
-      waitTaskState: [:]
+      waitTaskState: [:],
```
[the remark this answers is not in the document]
Comment (emjburns) [reply orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/rollingpush/CheckForRemainingTerminationsTask.groovy]: removed

**Comment by emjburns** [comment]
@ajordens added a test and verified locally that this fixes the problem. Couldn't use `Instant.MIN` because of an SQL overflow. Using `Instant.EPOC` instead.

**Comment by ncknt** [comment]
Hey @emjburns, it looks like the commit above contains `javafx.util.Duration` in CheckForRemainingTerminationsTask.java. It's not used and it breaks the compilation.

**Comment by emjburns** [comment]
@ncknt arg I thought I removed that :( will fix. Sorry.

**Pull request merged**